tests/ui/loop-match/invalid.rs

```diff
@@ -142,6 +142,23 @@ fn break_without_value_unit() {
     }
 }
 
+fn break_without_label() {
+    let mut state = State::A;
+    #[loop_match]
+    'a: loop {
+        state = 'blk: {
+            match state {
+                _ => {
+                    #[const_continue]
+                    break State::A;
+                    //~^ ERROR unlabeled `break` inside of a labeled block
+                    //~| ERROR mismatched types
```

Comment:
Where is mismatched types coming from?..

Reply:
can you fix the error so that the only errors here are the interesting ones? (e.g. by doing `_ = loop {...};`) or do you think this test is better with the type error?